src/screens/Auth/LoginScreen.tsx

```diff
@@ -12,6 +13,21 @@ import * as Yup from 'yup';
 import AntIcon from 'react-native-vector-icons/AntDesign';
 import {useKakaoAuthActions, useLocalAuthActions} from 'recoil/actions';
 
+GoogleSignin.configure({
+  scopes: ['https://www.googleapis.com/auth/userinfo.email'], // what API you want to access on behalf of the user, default is email and profile
+  webClientId:
+    '452065527838-h0oastn5pmuhtn32o72j3vjihjjn8qvh.apps.googleusercontent.com', // client ID of type WEB for your server (needed to verify user ID and offline access)
+  offlineAccess: true, // if you want to access Google API on behalf of the user FROM YOUR SERVER
+  // hostedDomain: '', // specifies a hosted domain restriction
+  // forceCodeForRefreshToken: true, // [Android] related to `serverAuthCode`, read the docs link below *.
+  // accountName: '', // [Android] specifies an account name on the device that should be used
+  iosClientId:
+    '452065527838-h0oastn5pmuhtn32o72j3vjihjjn8qvh.apps.googleusercontent.com', // [iOS] if you want to specify the client ID of type iOS (otherwise, it is taken from GoogleService-Info.plist)
+  // googleServicePlistPath: '', // [iOS] if you renamed your GoogleService-Info file, new name here, e.g. GoogleService-Info-Staging
+  // openIdRealm: '', // [iOS] The OpenID2 realm of the home web server. This allows Google to include the user's OpenID Identifier in the OpenID Connect ID token.
+  profileImageSize: 120, // [iOS] The desired height (and width) of the profile image. Defaults to 120px
+});
+
```

Comment:
구글 로그인할때 구성을 이렇게 하는군요!  
처음에만 초기 구성할거라면 useEffect로 한번만 실행시켜도 나쁘지 않을거 같아요 :)

```javascript
import React, {useRef, useState, useEffect} from 'react';

export default function LoginScreen() {

  useEffect(() => {
 GoogleSignin.configure({
  scopes: ['https://www.googleapis.com/auth/userinfo.email'], // what API you want to access on behalf of the user, default is email and profile
  webClientId:
    '452065527838-h0oastn5pmuhtn32o72j3vjihjjn8qvh.apps.googleusercontent.com', // client ID of type WEB for your server (needed to verify user ID and offline access)
  offlineAccess: true, // if you want to access Google API on behalf of the user FROM YOUR SERVER
  // hostedDomain: '', // specifies a hosted domain restriction
  // forceCodeForRefreshToken: true, // [Android] related to `serverAuthCode`, read the docs link below *.
  // accountName: '', // [Android] specifies an account name on the device that should be used
  iosClientId:
    '452065527838-h0oastn5pmuhtn32o72j3vjihjjn8qvh.apps.googleusercontent.com', // [iOS] if you want to specify the client ID of type iOS (otherwise, it is taken from GoogleService-Info.plist)
  // googleServicePlistPath: '', // [iOS] if you renamed your GoogleService-Info file, new name here, e.g. GoogleService-Info-Staging
  // openIdRealm: '', // [iOS] The OpenID2 realm of the home web server. This allows Google to include the user's OpenID Identifier in the OpenID Connect ID token.
  profileImageSize: 120, // [iOS] The desired height (and width) of the profile image. Defaults to 120px
});
},[])

(...)
```